packages/boot/test/bootstrapTests/test-orchestration.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable camelcase */
```

Comment:
it's not obvious why `camelcase` should be disabled. is it easy to add a few words? no big deal for test code, I guess